notatex-app/src/test/java/com/example/notatex/stepDefinition/ManageAccounts.java

```diff
@@ -0,0 +1,72 @@
+package com.example.notatex.stepDefinition;
+
+import com.example.notatex.website.NotatexPage;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import io.github.bonigarcia.wdm.WebDriverManager;
```

Comment:
Btw in selenium 4.6.0 it can be entirely skipped and manager is now embedded into the framework (although still in beta)

Reply:
<img width="556" alt="Zrzut ekranu 2022-12-5 o 20 10 23" src="https://user-images.githubusercontent.com/77123304/205722482-3d9fa492-4887-42ba-b59d-4eb3dd83d7ba.png">